### DISCUSSION TITLE
How to set slider value programatically

### DISCUSSION THREAD

**Discussion by bglima** [bulletphysics/bullet3#3697] [General]
Hello. Would there be a way to set DebugVariables current value programmatically?

I checked the docs and found binds for addUserDebugParameter and readUserDebugParameter related to user GUI. Is it possible for me to change settings (such as current value and limits) for an already existing debug parameter? Something like a setUserDebugParameter?

**Comment by thume4zzzz** [comment]
Hey, have you found the solution to set the value?
